hw2/03-count.js

```diff
@@ -1,3 +1,26 @@
 // Add your code here
+const input = document.getElementById("userInput");
 
-input.addEventListener('keydown', handleKeyDown);
+function handleKeyDown(event) {
+  const textDiv = document.getElementById("paragraph");
+  const textList = textDiv.textContent.split(" ");
+  const userQuery = input.value.toLowerCase();
+
+  if (userQuery === "") {
+    textDiv.innerHTML = textDiv.textContent;
+    return;
+  }
+
+  const highlightedTextList = textList.map((text) => {
```

Comment:
I feel you can try using regex here, so that when you type any word it will not highlight the word which is part of other word too.


<img width="1151" alt="Screenshot 2024-04-24 at 6 38 23 PM" src="https://github.com/Norcrossover/CS464P-homework/assets/41821284/abf46d25-358f-4e20-b8d8-aee564fc0d8e">

Reply:
Good catch! I am not that great with Regex, but I'll look into it